src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/VacancyClient.cs

```diff
@@ -244,9 +245,9 @@ public Task<IEnumerable<string>> GetEmployerIdentifiersAsync(string userId)
             return _employerAccountProvider.GetEmployerIdentifiersAsync(userId);
         }
 
-        public Task<bool> GetEmployerEOIAsync(string employeraccountId)
+        public Task<AccountDetailViewModel> GetEmployerAccountDetailsAsync(string employeraccountId)
```

Comment:
`AccountDetailViewModel` is the wrong name from the type declared within the EmployerAccount client and types packages. It isn't a ViewModel, as such and for other reasons we should not let these types bleed into other bits of our codebase outside f the Provider class. Can you make the reference to type with the name `AccountDetailViewModel` only be contained within the `EmployerAccountProvider` class and map the object into a type we declare which is named `EmployerAccount` please. There is a bit of a mix and match and a need to refactor further particularly the `LegalEntityViewModel` which is a similar issue. However the best thing we can do now is not add more.

Reply:
Done